qubes/tests/integ/network.py

```diff
@@ -95,17 +109,23 @@ def _run_cmd_and_log_output(self, vm, cmd):
         """Used in tearDown to collect more info"""
         if not vm.is_running():
             return
-        with contextlib.suppress(subprocess.CalledProcessError):
-            output, _ = self.loop.run_until_complete(
-                vm.run_for_stdio(cmd, user="root", stderr=subprocess.STDOUT)
-            )
-            self.log.critical("{}: {}: {}".format(vm.name, cmd, output))
+        try:
+            if vm.klass == "AdminVM":
+                proc = subprocess.run(cmd, capture_output=True, check=True)
```

Comment:
different command format depending on target is confusing... maybe add `shell=True` for consistency?